apps/web-client/src/lib/db/cr-sqlite/note.ts

```diff
@@ -361,19 +370,27 @@ export async function updateNoteTxn(
 		return;
 	}
 
+	// We're deleting the existing txn and placing the new one instead of it
+	// in case of mere update, we're keeping the original `updated_at`
```

Comment:
I think we should have `updated_at` get a new value on update. Or change its name to `created_at`, if we really want this behavior.

Reply:
Regarding the `updated_at` / `last_bubbled_at`, I say: let's merge this, as #677 depends on it, and, I would much rather have it in (with working UI, accurate tests and all) before I can be sure I didn't skip / mess something up in the process. I've opened an issue: #690